### PR TITLE
ICDS - Update Aggregation and View for Dashboard v2

### PR DESCRIPTION
**Pull request by sheelio**
@emord 

This does a few things:
- Adds new columns to the existing aggregate tables
- Updates the views for those aggregate tables
- Adds a new aggregate table (for daily tracking)

One open TODO is purging any daily tables > 15 days old.  Not sure the best way to do that.

### CODE REVIEW

**Comment by sheelio** [comment]
@emord 
It looks like @kkrampa has added a migration to specify views that are already present in the aggregation scripts (they're on master, not in this PR).  Is that going to create issues with the views that are being manually created?

**Comment by emord** [comment]
@sheelio do you mean these? https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/migrations/0016_aggawcmonthly_aggccsrecordmonthly_aggchildhealthmonthly_aggdailyusageview_aggthrmonthly_awclocation_.py

It looks like managed = False so it should be fine https://docs.djangoproject.com/en/1.11/ref/models/options/#managed

File: custom/icds_reports/migrations/0016_add_v2_indicators.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from corehq.sql_db.operations import RawSQLMigration
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates'))
+
+
+class Migration(migrations.Migration):
```
Comment (emord) [inline custom/icds_reports/migrations/0016_add_v2_indicators.py]: You'll probably need to rebase branch this onto master and rename this migration and change the dependency

**Review by emord — CHANGES_REQUESTED** [review]
I think I was wrong actually. Since you're add columns you'll probably want to change the model definitions that he wrote. and make another migration

**Pull request closed without merging**